### PR TITLE
Complete Day 4

### PR DESCRIPTION
**Pull request by arashnrim**
Hey!

This PR will implement my page for Day 4. I hope that it's acceptable!

Thanks for putting together this fun project for us to work together in a rather unique way. Everyone has done excellent so far, and I'm glad to have put mine too. I can't wait to see what the story might morph into with future pages; who knows what might happen?

Thank you! 🙌

### CODE REVIEW

File: src/pages/4/index.html [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+            showWindow(vault)
+        }
+    </script>
+</head>
+
+<body class="bg-[#f4f7f8]">
+    <img src="mewmew.jpg" loading="lazy" class="h-screen w-screen object-contain" />
+    <main class="h-screen flex flex-col absolute justify-center items-center top-0" id="windows">
+        <section class="w-screen flex-grow grid grid-cols-12 grid-rows-5">
+            <button type="button" onclick="openPage('../1/index.html', 'cuppa')">
```
Comment (maggie-j-liu) [inline src/pages/4/index.html:168]:
```suggestion
            <button type="button" onclick="openPage('../1/index.html', 'metaverse')">
```

File: src/pages/4/index.html [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+                <div
+                    class="m-5 p-2 flex flex-col justify-center items-center text-center hover:bg-black/10 transition-colors rounded-lg">
+                    <img src="metaverse.svg" />
+                    <p>metaverse</p>
+                </div>
+            </button>
+            <button type="button" onclick="openPage('../2/index.html', 'cuppa')">
```
Comment (maggie-j-liu) [inline src/pages/4/index.html:175]:
```suggestion
            <button type="button" onclick="openPage('../2/index.html', 'braingate')">
```

**Review by maggie-j-liu — APPROVED** [review]
thank you! i'll merge this for now, but if you think these suggestions are correct you can make a new pr

**Comment by arashnrim** [comment]
Ooh, good catch! I might have overlooked those, thanks for point them out for me! I'll make the appropriate changes and a PR when changes have been made.

**Pull request merged**